.gitignore

```diff
@@ -4,7 +4,6 @@
 # Ignore some folders
 node_modules
 /static/framework
-dist
```

Comment:
For now, we should only have one `.gitignore` at the root of the project, with `dist` or `/dist`, I can accept both